version.sbt

```diff
@@ -1 +1,2 @@
-version in ThisBuild := "3.0.2-SNAPSHOT"
+// replaced by pipeline builds via set-version.sh
+version in ThisBuild := "3.0.2-local"
```

Comment:
Is it expected to have local version?

Reply:
reverted to -SNAPSHOT to keep it maven-compatible